util/http/http.go

```diff
@@ -30,7 +30,7 @@ const (
 
 // max number of chunks a cookie can be broken into. To be compatible with
 // widest range of browsers, you shouldn't create more than 30 cookies per domain
-var maxCookieNumber = env.ParseNumFromEnv(common.EnvMaxCookieNumber, 20, 0, math.MaxInt64)
+var maxCookieNumber = env.ParseNumFromEnv(common.EnvMaxCookieNumber, 20, 0, math.MaxInt32)
```

Comment:
We should probably do `math.MaxInt` instead, it should automatically determine the max value depending on system architecture. Also `ParseNumFromEnv` limits the number to be in `(math.MinInt, math.MaxInt)` so this is should safe to do.

```suggestion
var maxCookieNumber = env.ParseNumFromEnv(common.EnvMaxCookieNumber, 20, 0, math.MaxInt)
```